lib/fastlane/plugin/wpmreleasetoolkit/helper/git_helper.rb

```diff
@@ -124,7 +124,18 @@ def self.list_local_tags(matching: '*')
       # @param [Bool] delete_on_remote If true, will also delete the tag from the remote. Otherwise, it will only be deleted locally.
       #
       def self.delete_tags(tag_names, delete_on_remote: false)
-        Action.sh('git', 'tag', '-d', *tag_names)
+
+        tag_names = [tag_names] unless tag_names.is_a? Array
```

Comment:
Because we accept both arrays and strings, we need to always have an array to continue

Reply:
```suggestion
        tag_names = Array(tag_names)
```
Should work both if tag_names is already an Array or not – [See this SO post](https://stackoverflow.com/a/18358925)

Reply:
Ah nice - addressed in d3d0d673deebb9a288050efa0da9e92bdb279403